src/styles/components/map-explorer.styl

```diff
@@ -0,0 +1,22 @@
+.map-explorer
+  width: 100%
+  overflow: hidden
+  display: flex
+  position: relative
+  
+  .map-visuals
+    flex: 0 0 auto
+    width: 100%
+    height: 100%
+    
+    #mapVisuals
```

Comment:
Native Leaflet allows us to create a map either with an element's ID, or a reference to the HTML element itself. (http://leafletjs.com/reference-1.2.0.html#map-factory)

IIRC the reason I use IDs instead of an element reference is because the standard React ref method returns a React Element, which I'd then need to unpack into a standard HTML DOM Element, which I felt was messy. That, it's worth for me to re-check this assumption in the `-v2` branch.

Also: I should rename those `map-explorer.styl` classes anyway to keep those naming conventions consistent.